internal/backends/linuxkms/fullscreenwindowadapter.rs

```diff
@@ -108,13 +109,19 @@ impl FullscreenWindowAdapter {
                 self.rotation,
                 &|item_renderer| {
                     if let Some(mouse_position) = mouse_position.get() {
+                        let cursor_image = mouse_cursor_image();
                         item_renderer.save_state();
                         item_renderer.translate(
                             i_slint_core::lengths::logical_point_from_api(mouse_position)
                                 .to_vector(),
                         );
                         item_renderer.draw_image_direct(mouse_cursor_image());
                         item_renderer.restore_state();
+                        let cursor_rect = LogicalRect::new(
+                            euclid::point2(mouse_position.x, mouse_position.y),
+                            euclid::Size2D::from_untyped(cursor_image.size().cast()),
+                        );
+                        self.renderer.as_core_renderer().mark_dirty_region(cursor_rect.into());
```

Comment:
I think this is harmless, but I can also remove it if desired. It's a no-op as there won't be a mouse with the sw renderer, and otherwise mark_dirty_region is a no-op with the GPU renderers.